client/src/pages/Projects.js

```diff
@@ -57,13 +71,12 @@ const Projects = () => {
     }
 
     const deleteProject = (project) => {
-        let newProjects = [];
-        for (let index = 0; index < allProjects.length; index++) {
-            if(allProjects[index].name !== project.name){
-                newProjects.push(allProjects[index]);
-            }
-        }
-        setAllProjects(newProjects)
+        let id = project._id;
+        api().delete("/projects/" + id).then((response) => {
```

Comment:
I think that in the path, after id, we also need to have a forward slash '/'
`api().delete("/projects/" + id+"/").then((response) => {`
In postman that was the only way the request would go through.